### PR TITLE
Add support for Suse

### PR DESCRIPTION
**Pull request by michaelimfeld**
##### SUMMARY
Add support for Suse

##### ISSUE TYPE
 - Feature Pull Request

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -10,6 +10,11 @@ icinga2_agent_apt:
   repo: "deb http://packages.icinga.com/{{ ansible_distribution|lower }} icinga-{{ ansible_distribution_release }} main"
   key: "http://packages.icinga.com/icinga.key"
 
+# The suse repository and key from Icinga
+icinga2_agent_suse:
+  repo: "https://packages.icinga.com/SUSE/ICINGA-release.repo"
+  key: "http://packages.icinga.com/icinga.key"
```
Comment (eyenx) [inline defaults/main.yml:16]: is the `key` even needed if we use `auto_import_keys` in the tasks?

File: defaults/main.yml [hunk 2]
```diff
@@ -10,6 +10,11 @@ icinga2_agent_apt:
...
+
 # The apt repository for stretch-backports
 stretch_backports_apt:
```
Comment (eyenx) [inline defaults/main.yml:19]: also this variable isn't prefixed as `icinga2_` and not following our best practices. Should be fixed in a different PR.

**Review by fujexo — APPROVED** [review]
